Source/Core/Common/SymbolDB.h

```diff
@@ -42,6 +43,7 @@ struct Symbol
 	{}
 
 	std::string name;
+	std::unordered_map<char, std::string> comments;
```

Comment:
That `char` key looks a bit off, especially since you index based on `unsigned int` inside `SymbolDB::GetCommentFromAddr`